tests/e2e/env/utils/test-config.js

```diff
@@ -17,9 +17,14 @@ const getTestConfig = () => {
 	const rawTestConfig = fs.readFileSync( testConfigFile );
 
 	let testConfig = JSON.parse(rawTestConfig);
-	testConfig.baseUrl = testConfig.url.substr(0, testConfig.url.length - 1);
 	let testPort = testConfig.url.match(/[0-9]+/);
```

Comment:
Since we're going to be testing against websites that aren't just the local environment I think we should drop this regex solution in favor of the `URL` object. This is less a blocker and more just something to consider.

Reply:
@ObliviousHarmony The line that is deleted might be the reason for your previous comment.

Reply:
I believe so, as this removes the trailing slash from the `baseUrl`.

Reply:
@ObliviousHarmony I updated this to always remove the trailing slash.